cudapoa/include/claragenomics/cudapoa/utils.hpp

```diff
@@ -0,0 +1,161 @@
+/*
+* Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#pragma once
+
+#include "../../../src/allocate_block.hpp" // for estimate_max_poas()
```

Comment:
@r-mafi I think we would need to move the implementation into a cpp file to hide this include, otherwise this is leaking into the public header. Once the implementation is in cpp, this include will move to the cpp file as well